docsite/docs/cli/fal scripts/functions-and-variables.md

```diff
@@ -63,3 +63,28 @@ from sqlalchemy.types import Integer
 # Can be useful if data has `None` values
 write_to_source(df, 'source', 'table', dtype={'value': Integer()})
 ```
+
+### `airbyte_sync` and `fivetran_sync` functions
+
+These functions let you run sync jobs on respective [Airbyte](https://docs.airbyte.com/) and [Fivetran](https://fivetran.com/docs/getting-started) connections and connectors. API connection details as well information on connectors have to be provided in [`profiles.yml`](../Project%20Configs/Credentials#exctract-load-configuration).
```

Comment:
1. It might be confusing to see `name` is referred to as `config_name` in the helper functions. Should we clarify that?
2. Same thing with connection name and connector name. Might be good to clarify exactly what they are referring to in the YAML.

Reply:
done